src/pages/Profile/Section/ProfileSection.tsx

```diff
@@ -1,33 +1,83 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
 import { Avatar, Typography } from '@mui/material';
 import { useRecoilValue } from 'recoil';
-import { useGetProfileQuery } from '@api/memberApi';
+import { FollowInfo } from '@api/dto';
+import { useGetProfileQuery, useFollowMemberMutation, useUnFollowMemberMutation } from '@api/memberApi';
 import memberState from '@recoil/member.recoil';
 import OutlinedButton from '@components/Button/OutlinedButton';
+import TextButton from '@components/Button/TextButton';
+import FollowList from './FollowList';
 
 const ProfileSection = () => {
-  const userInfo = useRecoilValue(memberState);
-  const { data: profileInfo } = useGetProfileQuery(userInfo?.memberId || 0);
+  const [followState, setFollowState] = useState('none');
+
+  const { memberId } = useParams();
+  const myMemberId = useRecoilValue(memberState)?.memberId;
+  const otherMemberId = Number(memberId) || 0;
+
+  const { data: profileInfo } = useGetProfileQuery(otherMemberId);
+  const { mutate: FollowMember } = useFollowMemberMutation(otherMemberId);
+  const { mutate: UnFollowMember } = useUnFollowMemberMutation(otherMemberId);
+
+  const followInfo: { [key: string]: { state: string; list: FollowInfo[] } } = {
+    follower: { state: '팔로워', list: profileInfo?.follower || [] },
+    following: { state: '팔로잉', list: profileInfo?.followee || [] },
+    none: { state: '', list: [] },
+  };
+
+  const isFollowed = () => {
+    return profileInfo?.follower.some((follower) => follower.id === myMemberId);
+  };
+
+  const handleFollowStateButtonClick = (state: string) => {
+    if (followState === state) setFollowState('none');
+    else setFollowState(state);
+  };
+
+  const handleFollowButtonClick = () => {
+    if (isFollowed()) UnFollowMember();
+    else FollowMember();
+  };
 
   return (
     <div className="flex w-full flex-col space-y-8 p-4">
-      <div className="m-4">
-        <Avatar
-          className="!h-64 !w-64 !bg-subBlack !text-white"
-          alt="profile thumbnail"
-          src={userInfo?.thumbnailPath || ''}
-        />
-      </div>
+      <Avatar
+        className="m-4 !h-64 !w-64 !bg-subBlack !text-white"
+        alt="profile thumbnail"
+        src={profileInfo?.thumbnailPath || ''}
```

Comment:
여기 아마 서버 url로 넣어야 조회 가능할 거에요!
```suggestion
        src={profileInfo?.thumbnailPath ? getServerImgUrl(profileInfo?.thumbnailPath) : undefined}
```
(src 빈스트링으로도 기본이미지 출력 되나요?~? 저는 undefined로 처리했었어서 위에 예시 undefined 넣었긴 한데 빈스트링도 기본 이미지 출력 값으로 적용되는 값이면 꼭 undefined로 반영하실 필요는 없을 것 같습니다!)

test1 계정에 프로필 변경해뒀는데 한 번 테스트 해보시면 될 것 같아요!

Reply:
alt 넣으면 fallback으로 alt 첫글자 떠서 이미지 못받아 왔을 때 기본 아바타 이미지 띄워주려면 alt 적용 안 해야 할거에욧..!